docker/resource_docker_container_v1.go

```diff
@@ -789,6 +789,16 @@ func resourceDockerContainerV1() *schema.Resource {
 				Elem:        &schema.Schema{Type: schema.TypeString},
 				Set:         schema.HashString,
 			},
+			"tty": {
+				Type:     schema.TypeBool,
+				Default:  false,
+				Optional: true,
+			},
+			"stdin_open": {
+				Type:     schema.TypeBool,
+				Default:  false,
+				Optional: true,
+			},
```

Comment:
This function is used only for schema migration, so I think we don't have to update this function.

```suggestion
```

Reply:
okay, removed as well